backend/core/src/migration/1640878095625-favoriteID.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class favoriteID1640878095625 implements MigrationInterface {
+  name = "favoriteID1640878095625"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "user_preferences" ADD "favorite_i_ds" text array NOT NULL`
```

Comment:
To be consistent with existing cases, like `listingId` can you please update this to be `favorite_ids` and every where that has `favoriteIDs` to `favoriteIds`.